src/Paramol/SqlClient/SqlClientSyntax.DataTypes.cs

```diff
@@ -222,9 +222,9 @@ public IDbParameterValue DateTimeOffset(DateTimeOffset? value)
         /// <param name="precision">The parameter precision.</param>
         /// <param name="scale">The parameter scale.</param>
         /// <returns>A <see cref="IDbParameterValue" />.</returns>
-        public IDbParameterValue Decimal(decimal? value, int precision, int scale)
+        public IDbParameterValue Decimal(decimal value, byte precision, byte scale)
```

Comment:
Null is still desired in this case (think of a nullable decimal column). I prefer symmetry with all the other datatypes in that regard. That'll probably imply the introduction of `TSqlDecimalNullValue`.